### PR TITLE
British English-ing saturday.tex

### PR DESCRIPTION
**Pull request by jnicho02**
Small tweaks

### CODE REVIEW

File: saturday.tex [hunk 1]
```diff
@@ -51,10 +51,10 @@ \section*{Saturday Sessions Descriptions}\label{saturday-descriptions}
 }{%
   Karen M. Sandler is the executive director of the Software Freedom Conservancy, which is the
   nonprofit home of dozens of free and open source software projects including Git, Etherpad, QEMU,
```
Comment (boothym) [inline saturday.tex]: I think nonprofit should be hyphenated?

**Comment by jnicho02** [comment]
Think it's okay https://en.m.wikipedia.org/wiki/Nonprofit_organization

**Comment by boothym** [comment]
That wiki page seems like it's using American English. The OSM wiki page, plus the OSM UK website both have the hyphen, which I assumed was a British thing.

File: saturday.tex (hunk 1 above)
Comment (jnicho02) [reply saturday.tex]: I've added #18 so that it is dealt with in a future pull request

**Comment by jnicho02** [comment]
fixes #15

**Pull request merged**